Model/Response/Buyers.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Balancepay\Balancepay\Model\Response;
+
+use Balancepay\Balancepay\Model\AbstractResponse;
+
+/**
+ * Balancepay webhooks response model.
+ */
+class Buyers extends AbstractResponse
```

Comment:
Is this related to the webhook? if so, consider also renaming to something like `BuyersTopicWebhookResponse`

Reply:
This is not related to only webhook request, we also call differenct buyer related APIs using this